crates/sui-graphql-rpc/src/server/builder.rs

```diff
@@ -427,7 +428,8 @@ impl ServerBuilder {
             .context_data(zklogin_config)
             .context_data(metrics.clone())
             .context_data(config.clone())
-            .context_data(move_registry_config.clone());
+            .context_data(move_registry_config.clone())
```

Comment:
This is not something that we need to address in this PR, but as a future clean-up we need to standardise how we deal with configs in context data. Today we add the entire `ServiceConfig` to the context data, and then we separately add some of its constituent parts (`zklogin_config`, `name_service_config`, and now `move_registry_config`) -- not clear which way is better (splitting up `ServiceConfig` or not), but having both methods of accessing these configs is going to get confusing.

Reply:
Fair argument and we should indeed figure out a more reusable way of handling these.